src/theme/material/list-item.m.css

```diff
@@ -17,4 +18,30 @@
 .active {
 	border: none;
 	background: var(--mdc-theme-on-surface);
+	height: 48px;
+	box-shadow: none;
+}
+
+.collapsed {
```

Comment:
is it possible to remove the `important` usage here by adding a `.root` selector before `.collapsed` instead?